server/processor.go

```diff
@@ -111,19 +136,23 @@ func (p *PostProcessor) queuePostForProcessing(api plugin.API, post *model.Post)
 	}
 }
 
-func (p *PostProcessor) moderatePost(api plugin.API, post *model.Post) error {
-	if !p.shouldModerateUser(post.UserId) {
+func (p *PostProcessor) moderatePost(api plugin.API, post *model.Post, auditRecord *model.AuditRecord) error {
+	if post.Message == "" {
 		return nil
 	}
 
-	if !p.shouldModerateChannel(post.ChannelId) {
+	if !p.shouldModerateUser(post.UserId) {
```

Comment:
Would it provide value to also show why it was excluded?

Reply:
done :+1: